umbral-pre/Cargo.toml

```diff
@@ -10,7 +10,7 @@ readme = "README.md"
 categories = ["cryptography", "no-std"]
 
 [dependencies]
-k256 = { version = "0.10.1", default-features = false, features = ["ecdsa", "arithmetic"] }
+k256 = { git = "https://github.com/RustCrypto/elliptic-curves", default-features = false, features = ["ecdsa", "arithmetic", "hash2curve"] }
```

Comment:
Should we add a revision to `git` configuration in order to avoid breaking changes etc.?

Reply:
This is just temporary to make the CI run, I'm not planning on merging this until k256 0.10.2 is released